downstream/modules/aap-hardening/ref-interactive-session-timeout.adoc

```diff
@@ -0,0 +1,24 @@
+[id="ref-interactive-session-timeout"]
+
+=  Interactive session timeout
+
+A compliance profile might require that an interactive session timeout be enforced. 
+For example, the DISA STIG requires that all users be automatically logged out after 15 minutes of inactivity. 
+The installation process often requires an hour or more to complete, and this control can stop the installation process and log out the user before installation is complete. 
+The same also applies to day two operations such as backup and restore, which in production environments often take longer than the recommended interactive session timeout. 
+During these operations, the interactive session timeout should be increased to ensure the operation is successful.
```

Comment:
To avoid the use of "should" (per the style guide), perhaps say something like "...For these operations, increase the interactive session timeout to ensure the operation is successful." This might also make it more of an active voice.